### ISSUE TITLE
since 1.5.0 `USE_DJANGO_JQUERY` is False by default, in constrast to what is claimed by README and previous versions

### ISSUE THREAD

**Issue by rooterkyberian** [digi604/django-smart-selects#206]
Since 1.5.0 `USE_DJANGO_JQUERY` is False by default, in contrast to what is claimed by README and how previous versions behaved. 

The discrepancy between README and actual behavior makes me believe this change was unintended.

## Checklist

- [x] This issue is not about installing previous versions of django-smart-selects older than 1.2.8. I understand that previous versions are insecure and will not receive any support whatsoever.
- [x] I have verified that that issue exists against the `master` branch of django-smart-selects.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] I have debugged the issue to the `smart_selects` app.
- [x] I have reduced the issue to the simplest possible case.
- [ ] I have included all relevant sections of `models.py`, `forms.py`, and `views.py` with problems.
- [ ] I have used [GitHub Flavored Markdown](https://help.github.com/articles/creating-and-highlighting-code-blocks/) to style all of my posted code.

## Steps to reproduce
use default settings
## Expected behavior
jquery used is the one included with Django as explained by the readme
## Actual behavior
by default jquery will be fetched from `ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js`
to change this both `USE_DJANGO_JQUERY` and `JQUERY_URL` have to be modified:
```py
USE_DJANGO_JQUERY = True
JQUERY_URL = None
```
and only then jquery bundled with Django will be used

Claim in README:
https://github.com/digi604/django-smart-selects/blob/1.5.1/README.md#settings
> USE_DJANGO_JQUERY : By default, smart_selects will use the bundled jQuery from Django 1.2's admin area. Set USE_DJANGO_JQUERY = False to disable this behaviour.

This behavior was changed in 1.5.0:
> https://github.com/digi604/django-smart-selects/compare/1.4.0...1.5.0#diff-7bd64618692ef6f50012e6d8dbc251e7R17
and although this line received a fix in 1.5.1, default still remains is `False` where as both in past versions (<1.5.0) and in README is claimed to be `True`
https://github.com/digi604/django-smart-selects/compare/1.5.0...1.5.1#diff-7bd64618692ef6f50012e6d8dbc251e7R17

Situation is even more complicated, because for django > 1.2 the `JQUERY_URL` was always set to `None` by default, without which the `USE_DJANGO_JQUERY` doesn't even mean anything due how conditions here are constructed: https://github.com/digi604/django-smart-selects/blob/1.5.1 /smart_selects/widgets.py#L31

**Comment by rooterkyberian** [comment]
Note how I stumbled on this issue:

For me with 1.5.0 and 1.5.1 defaults I got regression caused by javascript error `ReferenceError: django is not defined`. I don't have now the time to debug the cause of this, just putting it in here if some lost soul from google encounters similar issue after updating.
Workaround was to return previous default by setting
```
USE_DJANGO_JQUERY = True
JQUERY_URL = None
```
explicitly in settings file.
